src/software/field_tests/field_test_fixture.py

```diff
@@ -541,13 +553,19 @@ def field_test_runner():
         multicast_channel=getRobotMulticastChannel(0),
         interface=args.interface,
         disable_estop=False,
-    ) as rc_blue:
+    ) as rc_blue, RobotCommunication(
+        current_proto_unix_io=yellow_full_system_proto_unix_io,
+        multicast_channel=getRobotMulticastChannel(0),
+        interface=args.interface,
+        disable_estop=False,
+    ) as rc_yellow:
```

Comment:
As discussed offline, we should remove the second robot communication, Fullsystem, ProtoUnixIO, ProtoLogger, ...
We should also have a cmd line arg for configuring the color of the one fullsystem we have.

Reply:
removed